src/toolkit/metadatas.cc

```diff
@@ -0,0 +1,223 @@
+#include <math.h>
```

Comment:
Replace metatdatas with metadata throughout this file, including the filename. Same with Metadatas (should be Metadata).

Reply:
(and datas->data and METADATAS->METADATA)